src/scyjava/__init__.py

```diff
@@ -1098,6 +1111,7 @@ def _import_java_classes():
     List = jimport("java.util.List")
     Map = jimport("java.util.Map")
     Set = jimport("java.util.Set")
+    JavaPath = jimport("java.nio.file.Path")
```

Comment:
I suggest waiting till #52 is merged, then rebasing this PR on top. Then you could put the `java.nio.file.Path` import into the `_jc` container of `JavaClasses`, to avoid the name clash.